app/views/volunteers/_notes.html.erb

```diff
@@ -19,7 +19,7 @@
               <tr>
                 <td><%= note.content %></td>
                 <td><%= note.creator.display_name %></td>
-                <td><%= note.created_at.strftime(t("time.formats.standard")) %></td>
+                <td><%= note.created_at.strftime("%m-%d-%Y") %></td>
```

Comment:
Could we actually take advantage of the translation file for this? That way if we change the way we want to display the date, we only have to change it in one place:
```suggestion
                <td><%= l(note.created_at, format: :standard) %></td>
```

Ref: https://guides.rubyonrails.org/i18n.html#adding-date-time-formats